package.json

```diff
@@ -24,16 +24,15 @@
   "scripts": {
     "all-test": "npm run-script node-test && npm run-script smoke-test",
     "build": "ember build",
-    "client-test": "./run-smoke-test.sh all-commands.sh",
     "lint": "eslint lib test",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "node-test": "mocha test/**/*.js",
     "node-test-with-coverage": "nyc --reporter lcov npm run-script node-test && node_modules/.bin/codecov",
-    "smoke-test": "./run-smoke-test.sh smoke-test.sh",
-    "smoke-test-yarn": "./run-smoke-test.sh smoke-test-yarn.sh",
+    "smoke-test": "cd smoke-test-app && ./smoke-test.sh",
```

Comment:
I think these scripts are "weird" now locally because unless you've gone into `smoke-test-app` and installed with the correct package manager, you can easily run `smoke-test-yarn` and have it being using npm, and vice versa.

Reply:
I just decided to delete these scripts all together. I couldn't think of a way to ensure they actually run in the right contexts, and since CI is actually pretty fast / peppy it's _probably_ fine...